spec/modules/authenticate_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe Authenticate do
+  let(:user) { FactoryGirl.create(:user) }
+
+  class ExampleClass
```

Comment:
Nie wiem czy nie wygodniej będzie Ci dziedziczyć tej klasy po `ActionController::Base` - nie będziesz musiała definiować `helper_method` i ew. inne dziwne zachowania też mogą wyjść. Od razu zaznaczę że nie wiem czy nie spowoduje to problemów - mam nadzieję że nie, ale nie pisałem ostatnio takiego testu więc nie pamiętam.